phoenix-core/src/main/java/org/apache/phoenix/query/QueryServices.java

```diff
@@ -123,9 +123,10 @@
     // Index will be partially re-built from index disable time stamp - following overlap time
     public static final String INDEX_FAILURE_HANDLING_REBUILD_OVERLAP_TIME_ATTRIB =
         "phoenix.index.failure.handling.rebuild.overlap.time";
-    public static final String MIN_INDEX_PRIOIRTY_ATTRIB = "phoenix.regionserver.index.priority.min";
-    public static final String MAX_INDEX_PRIOIRTY_ATTRIB = "phoenix.regionserver.index.priority.max";
+    public static final String INDEX_PRIOIRTY_ATTRIB = "phoenix.regionserver.index.priority";
     public static final String INDEX_HANDLER_COUNT_ATTRIB = "phoenix.regionserver.index.handler.count";
+    public static final String METADATA_PRIOIRTY_ATTRIB = "phoenix.regionserver.metadata.priority";
```

Comment:
This is a client side and region server param, isn't it? How about phoenix.index.rpc.priority and phoenix.metadata.rpc.priority?